src/Build/Logging/DistributedLoggers/DistributedFileLogger.cs

```diff
@@ -97,6 +97,7 @@ public void Initialize(IEventSource eventSource)
             ErrorUtilities.VerifyThrowArgumentNull(eventSource, nameof(eventSource));
             ParseFileLoggerParameters();
             string fileName = _logFile;
+
```

Comment:
This seems like a very reasonable change to put in a PR that touches DistributedFileLogger in any other way...

Reply:
True, but also I think it's now or never :D